build.gradle

```diff
@@ -1058,25 +1080,35 @@ project('test:system') {
         systemProperty "bookkeeperImageVersion", System.getProperty("bookkeeperImageVersion", "imageVersion") // // bookkeeper image version
         // tier2Type , default is NFS.
         systemProperty "tier2Type", System.getProperty("tier2Type", "nfs")
-        // tier2 Configuration, specified as comma seperated key values k1=v1,k2=v2
-        systemProperty "tier2Config", System.getProperty("tier2Config")
-        systemProperty "pravegaOperatorVersion", System.getProperty("pravegaOperatorVersion", "latest")
-        systemProperty "bookkeeperOperatorVersion", System.getProperty("bookkeeperOperatorVersion", "latest")
-        systemProperty "zookeeperOperatorVersion", System.getProperty("zookeeperOperatorVersion", "latest")
-        systemProperty "desiredPravegaCMVersion", System.getProperty("desiredPravegaCMVersion", "0.9.0")
-        systemProperty "desiredBookkeeperCMVersion", System.getProperty("desiredBookkeeperCMVersion", "0.9.0")
-        systemProperty "publishedChartName", System.getProperty("publishedChartName", "pravega")
-        systemProperty "helmRepository", System.getProperty("helmRepository", "pravega")
+        // tier2 configuration, specified as comma separated key values k1=v1,k2=v2
+        systemProperty "tier2config", System.getProperty("tier2config")
```

Comment:
This change seems gratuitous and it breaks the camelCase style of the rest of parameters. Also, this could be breaking other Jenkins jobs that rely on the former property name.

Reply:
Right.. this change was not intentional.